### ISSUE TITLE
Cannot refresh timelines/notifications with TalkBack

### ISSUE THREAD

**Issue by charlag** [tuskyapp/Tusky#1711]
There's no way to perform refresh with a screen reader. At least I don't know any. I think it's clear how painful it is to use the app like that.

Google recommends having a menu item (although it's not about Talkback):
> You should add a refresh action to your app's action bar to ensure that users who may not be able to perform a swipe gesture can still trigger a manual update. For example, users with accessibility needs can trigger action bar actions using external devices, such as keyboards and D-pads.
You should add the refresh action as a menu item, rather than as a button, by setting the attribute android:showAsAction=never. If you display the action as a button, users may assume that the refresh button action is different from the swipe-to-refresh action. By making the refresh action less conspicuous in the action bar, you can encourage users to perform manual updates with the swipe gesture while still maintaining the accessible option in a place where D-pad users would look for it.

https://developer.android.com/training/swipe/add-swipe-interface

We could add it to the post a11y actions although I think it's weird.

What we could do is to rethink our layout and make it

```
 ≡      __search__        ⋮
----------------------------
 home | notifications | else
```
and make the top one disappear on scroll (that's almost exactly current Google Play layout)

* * * *
- Tusky Version: 10.0 beta 1-aba36ca
- Android Version: 10
- Android Device: Fairphone 3

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by charlag** [comment]
I was actually wrong and there *is* a way, with a second finger
https://megadon.net/@n0btc/103721776419828517
but it's not very ergonomic at best.

**Comment by nikclayton** [comment]
I noticed this same problem while I was looking at the thread fragment, so I'll take a go at fixing it by following the recommendation to add a menu item to the activities or fragments that need it.

Right now I think that's:

- TimelineFragment
- AccountMediaFragment
- ConversationsFragment
- ReportStatusesFragment
- ScheduledStatusActivity
- SearchFragment
- ViewThreadFragment
- NotificationsFragment

**Comment by connyduck** [comment]
https://github.com/tuskyapp/Tusky/pull/3121
